docker/docker/Dockerfile.linux.amd64

```diff
@@ -1,4 +1,15 @@
+FROM golang
+
+ENV GOOS="linux"
+ENV GOARCH="amd64"
+ENV CGO_ENABLED=0
+ENV GO111MODULE="on"
+
+WORKDIR /build
+ADD . .
+RUN go build -v -a -tags netgo -o release/linux/amd64/kaniko-docker ./cmd/kaniko-docker
+
 FROM gcr.io/kaniko-project/executor:v1.3.0
 
-ADD release/linux/amd64/kaniko-docker /kaniko/
+COPY --from=0 /build/release/linux/amd64/kaniko-docker /kaniko/
```

Comment:
We don't normally build the go binaries from dockerfile. There is a scripts/docker.sh file present which can be used to generate go binaries as well as docker build. This is followed by other plugins e.g. drone docker plugin https://github.com/drone-plugins/drone-docker/blob/master/docker/docker/Dockerfile.linux.amd64

Reply:
If this helps, probably we can close the pull request.